config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
Without this change, the E2E tests pass, but the manager Pod has warnings and only 1/2 container started:

```
sample-external-issuer-system   sample-external-issuer-controller-manager-7b7fd6fb69-9mg9w         1/2     CreateContainerConfigError   0          55s

```

```
  Normal   Pulled     11s (x9 over 93s)  kubelet, sample-external-issuer-e2e-control-plane  Container image "gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0" already present on machine
  Warning  Failed     11s (x9 over 93s)  kubelet, sample-external-issuer-e2e-control-plane  Error: container has runAsNonRoot and image will run as root

```